backend/src/app/modules/video/video.service.ts

```
@@ -9,16 +11,51 @@
+                    moduleItemId: video?.moduleItemId ?? ""
+                }
+            },
+            include: {
+                VideoProgress: true
+            }
+
+        })
+
+        result.progress = videoProgress?.VideoProgress ?? null
+    }
+    delete result.moduleItem.module
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)